src/requirements/data/university/index.ts

```diff
@@ -8,6 +8,7 @@ const universityRequirements: readonly CollegeOrMajorRequirement[] = [
       'All incoming freshmen are required to take two credits (two courses) of Physical Education, ' +
       'one credit each semester of the first year on campus.',
     source: 'http://courses.cornell.edu/content.php?catoid=41&navoid=11637',
+    allowCourseDoubleCounting: true,
```

Comment:
I don't know if it's just because I'm traumatized by CS4620, but I think allowCourseDoubleCounting can be a long name. If you think it belongs, it's fine.

Reply:
I tend to prefer this long name for clarify. Previously people seem to be confused whether the allow double counting is a concept on the requirement or the course, so I think extra long name can serve as a clarification.